src/_scss/pages/covid19/index.scss

```diff
@@ -170,10 +163,9 @@
 
         .heading__container {
           border-radius: 0.5rem 0.5rem 0 0;
-          @include display(flex);
+          @include display(block);
           flex-wrap: wrap;
```

Comment:
This should be removed; no longer a flex container

Reply:
@afnazu Did you see this line should be removed?

Reply:
@brett-varney Thanks, removed css rule and updated footer styling